CHANGELOG_PENDING.md

```diff
@@ -1,3 +1,9 @@
 # Unreleased Changes
 
 ## vX.X
+
+### Bug Fixes 🐞
+
+#### General
+
+- [#1809](https://github.com/livepeer/go-livepeer/pull/1809) Avoid nil pointer error in SyncToLatestBlock when no blocks are present in the database (@kyriediculous)
```

Comment:
Let's update this based on format description in the [contributing](https://github.com/livepeer/go-livepeer/blob/master/doc/contributing.md#changelog) guide.